bundle-workflow/src/run_integ_test.py

```diff
@@ -57,13 +59,26 @@ def main():
                     work_dir,
                     args.s3_bucket
                 )
-                test_suite.execute()
+                status, security = test_suite.execute()
+                if status != 0:
+                    failed_components[component.name] = [status, security]
+                else:
+                    passed_components[component.name] = [status, security]
             else:
                 logging.info(
                     "Skipping tests for %s, as it is currently not supported"
                     % component.name
                 )
 
+        if passed_components:
+            for component, result in passed_components.items():
+                logging.info(f'PASS: Integration Test for {component} with status code {result[0]} and config is {result[1]}')
+
+        if failed_components:
+            for component, result in failed_components.items():
+                logging.error(f'FAIL: Integration Test for {component} with status code {result[0]} and config is {result[1]}')
```

Comment:
I would like, it feels more cleaner.
```suggestion
                logging.error(f'FAIL: Integration Test for {component} {result[1]} with status code {result[0]} ')
```